Source/DafnyLanguageServer/Workspace/ProjectManager.cs

```diff
@@ -103,33 +107,25 @@ public class ProjectManager : IDisposable {
     options = DetermineProjectOptions(project, serverOptions);
     options.Printer = new OutputLogger(logger);
     boogieEngine = new ExecutionEngine(options, cache, scheduler);
-    var initialCompilation = GetCompilationInput();
+    var initialCompilation = new CompilationInput(options, version, Project);
```

Comment:
Here I see CompilationInput. What has changed exaclty? What is Compilation vs. CompilationInput?

Reply:
`CompilationInput` is all the inputs of a compilation, so the DafnyProject and the DafnyOptions that result from combining the CLI argument with the DafnyProject.

`Compilation` is the class responsible for doing most of what our codebase does:
1. Based on the project file and files passed to the CLI, determine what the root files for parsing are, and whether they are preverified or precompiled
1. Parse files
1. Resolve modules
1. Verify declarations
1. Not yet: translate the program to a target language